src/vtab/series.rs

```diff
@@ -28,7 +28,7 @@ const SERIES_COLUMN_STOP: c_int = 2;
 const SERIES_COLUMN_STEP: c_int = 3;
 
 bitflags::bitflags! {
-    #[derive(Clone, Copy)]
+    #[derive(Clone, Copy, Debug)]
```

Comment:
Use only to pass information between `best_index` and `filter` methods.

Reply:
reverted